plone-5.2.x.cfg

```diff
@@ -0,0 +1,5 @@
+buildout]
+extends =
+    base.cfg
+    http://dist.plone.org/release/5.2a1/versions.cfg
+find-links += http://dist.plone.org/thirdparty/
```

Comment:
On 5.1.x.cfg you extend versions.cfg while it is not extended here on 5.2.cfg. and the other way around with this find-links option.

It might be a good idea to keep the version specific configuration to a minimum, and put as much configuration on the shared configuration file (travis.cfg)